testsuite/do_tests.sh.in

```diff
@@ -797,6 +797,20 @@ else
     echo "  for Python. See the file README.md for details."
 fi
 
+echo
+echo "Phase 8: Running C++ tests"
+echo "--------------------------"
+
+CPP_TEST_OUTPUT="$(run_all_cpptests 2>&1)"
```

Comment:
When I run `make installcheck` on my local system I get the following error:
```
Phase 8: Running C++ tests
--------------------------
/home/hahne/git/5g.install/share/nest/extras/do_tests.sh: 804: /home/hahne/git/5g.install/share/nest/extras/do_tests.sh: run_all_cpptests: not found
/home/hahne/git/5g.install/share/nest/extras/do_tests.sh: 810: /home/hahne/git/5g.install/share/nest/extras/do_tests.sh: arithmetic expression: expecting primary: " 859 +  "

```

Reply:
me too. @hakonsbm do you know what could cause this?